### PR TITLE
Adds a full nun outfit to the Chaplain's lockers!

### PR DESCRIPTION
**Pull request by AnguishedEnglish**
## About the PR
Adds a complete Nun Outfit, complete with robes and a matching hood that appear in all of the Chaplain's personal lockers!

Nun Full New
Nun Dress Object
Nun Hat Sprite Object

## Why's this needed? 
Gives female chaplain's more clothing options which were a bit lacking.

## Changelog
```changelog
(u)AnguishedEnglish
(*)Added a Nun Robe and Nun Hood to the Chaplain's lockers.
```

### CODE REVIEW

File: code/__secret_public.dme [hunk 1]
```diff
@@ -14,3 +14,14 @@
...
+// END_INTERNALS
+// BEGIN_FILE_DIR
+#define FILE_DIR .
+// END_FILE_DIR
+// BEGIN_PREFERENCES
+// END_PREFERENCES
+// BEGIN_INCLUDE
+#include "_placeholder.dm"
+#include "_publicVersion.dm"
+// END_INCLUDE
```
Comment (Kyle2143) [inline code/__secret_public.dme:27]: This needs to be cleaned up.

Comment (ZeWaka) [reply code/__secret_public.dme:27]: due to editing with DM

Comment (pali6) [inline code/__secret_public.dme:27]:
```suggestion
```

**Review by pali6 — APPROVED** [review]

**Comment by pali6** [comment]
Probably should have a changelog entry.

**Comment by ZeWaka** [comment]
@AnguishedEnglish Changelog needs to follow the format as specified in the PR template

**Comment by AnguishedEnglish** [comment]
> @AnguishedEnglish Changelog needs to follow the format as specified in the PR template

I don't know where that is...

**Pull request merged**